app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(elves: list[Elf]) -> list[None]:
+    return [elf.play_elf_song() for elf in elves]
```

Comment:
Do not use List comprehensions if you do not intend to store the result in a variable. Use a regular for loop here.